src/components/NodeLink/NodeLink.vue

```diff
@@ -204,13 +198,31 @@ export default {
 </script>
 
 <template>
-  <svg
-    ref="svg"
-    width="800"
-    height="900"
-  />
+  <div>
+    <svg
+      ref="svg"
+      width="800"
+      height="900"
+    />
+    <div class="tooltip" />
+  </div>
 </template>
 
 <style scoped>
 @import './NodeLink.css';
+
+.tooltip {
+  position: absolute;
+  opacity: 0;
+  background-color: white;
+
+  font-size:12.5px;
+  color:#000;
+  border-radius:5px;
+  padding:5px;
+  pointer-events:none;
+  -webkit-box-shadow:0 4px 8px 0 rgba(0,0,0,.2);
+  box-shadow:0 4px 8px 0 rgba(0,0,0,.2);
+  max-width:400px
```

Comment:
A space after the colon would make this more readable (like you have in lines 215-217 above).

Also, is the vendor version of `box-shadow` needed? Maybe @jtomeck could weigh in.

Reply:
Fixed your first point in b178885